tsl/test/expected/chunk_utils_internal.out

```diff
@@ -119,12 +165,40 @@ FROM _timescaledb_catalog.chunk WHERE table_name = :'CHUNK_NAME';
  _hyper_1_1_chunk |      4
 (1 row)
 
+-- Check trigger
+\d :CHNAME
+       Table "_timescaledb_internal._hyper_1_1_chunk"
+ Column |       Type       | Collation | Nullable | Default 
+--------+------------------+-----------+----------+---------
+ time   | bigint           |           | not null | 
+ temp   | double precision |           |          | 
+Indexes:
+    "_hyper_1_1_chunk_hyper1_time_idx" btree ("time" DESC)
+Check constraints:
+    "constraint_1" CHECK ("time" >= '10'::bigint AND "time" < '20'::bigint)
+Triggers:
+    frozen_chunk_modify_blocker_row BEFORE INSERT OR DELETE OR UPDATE ON _timescaledb_internal._hyper_1_1_chunk FOR EACH ROW EXECUTE FUNCTION _timescaledb_internal.frozen_chunk_modify_blocker()
+    frozen_chunk_modify_blocker_stmt BEFORE TRUNCATE ON _timescaledb_internal._hyper_1_1_chunk FOR EACH STATEMENT EXECUTE FUNCTION _timescaledb_internal.frozen_chunk_modify_blocker()
+Inherits: test1.hyper1
+
 SELECT  _timescaledb_internal.unfreeze_chunk( :'CHNAME');
  unfreeze_chunk 
 ----------------
  t
 (1 row)
 
+\d :CHNAME
```

Comment:
Could you please select directly from pg_trigger instead of using a describe? SELECT tgname from pg_trigger WHERE oid = and tgname LIKE 'xxx';

Reply:
Initially, I tried to get the information from `information_schema.triggers` with the following statement: 

```sql
SELECT trigger_name, event_manipulation, action_statement, action_orientation, action_timing 
FROM information_schema.triggers WHERE trigger_catalog = 'db_chunk_utils_internal' AND
event_object_table = :'CHUNK_NAME'
ORDER BY event_object_table, trigger_name, event_manipulation;
```

Unfortunately, due to the [SQL standard](https://www.postgresql.org/message-id/16934.1568989957%40sss.pgh.pa.us), truncate triggers are not listed in this view.

```sql
test_300=# CREATE TABLE metric_name(id int primary key, name text);
test_300=# CREATE TRIGGER test1 BEFORE INSERT OR UPDATE OR DELETE ON metric_name EXECUTE FUNCTION _timescaledb_internal.frozen_chunk_modify_blocker();
CREATE TRIGGER
test_300=# CREATE TRIGGER test2 BEFORE TRUNCATE ON metric_name FOR STATEMENT EXECUTE FUNCTION _timescaledb_internal.frozen_chunk_modify_blocker();
CREATE TRIGGER

test_300=# SELECT trigger_name, event_manipulation, action_statement, action_orientation, action_timing FROM information_schema.triggers;
 trigger_name | event_manipulation |                           action_statement                           | action_orientation | action_timing 
--------------+--------------------+----------------------------------------------------------------------+--------------------+---------------
 test1        | INSERT             | EXECUTE FUNCTION _timescaledb_internal.frozen_chunk_modify_blocker() | STATEMENT          | BEFORE
 test1        | DELETE             | EXECUTE FUNCTION _timescaledb_internal.frozen_chunk_modify_blocker() | STATEMENT          | BEFORE
 test1        | UPDATE             | EXECUTE FUNCTION _timescaledb_internal.frozen_chunk_modify_blocker() | STATEMENT          | BEFORE
(3 rows)
```

As you suggested, we can get the output from `pg_trigger`, but I think it is hard to read.

```sql
test_300=# SELECT * from pg_trigger where tgrelid='metric_name'::regclass::oid;
  oid  | tgrelid | tgparentid | tgname | tgfoid | tgtype | tgenabled | tgisinternal | tgconstrrelid | tgconstrindid | tgconstraint | tgdeferrable | tginitdeferred | tgnargs | tgattr | tgar
gs | tgqual | tgoldtable | tgnewtable 
-------+---------+------------+--------+--------+--------+-----------+--------------+---------------+---------------+--------------+--------------+----------------+---------+--------+-----
---+--------+------------+------------
 29636 |   29628 |          0 | test1  |  29355 |     30 | O         | f            |             0 |             0 |            0 | f            | f              |       0 |        | \x  
   |        |            | 
 29637 |   29628 |          0 | test2  |  29355 |     34 | O         | f            |             0 |             0 |            0 | f            | f              |       0 |        | \x  
   |        |            | 
(2 rows)
```

We could try to decode the values like this:

```sql 
test_300=# SELECT tgname, text from pg_trigger t, ( VALUES (4,'INSERT'::text), (8,'DELETE'::text), (16,'UPDATE'::text), (32,'TRUNCATE'::text)) em(num, text) where tgrelid='metric_name'::regclass:
 tgname |   text   
--------+----------
 test1  | INSERT
 test1  | DELETE
 test1  | UPDATE
 test2  | TRUNCATE
(4 rows)
```
However, I think the `\d` statement is easier to understand. Do you prefer to decode the `pg_trigger` table manually?

Reply:
I don't think we need to decode. IMO, select tgname, tgtype from pg_trigger is good enough.

Reply:
It is changed in the newest version of the PR